src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapGroupBySettings.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Radio } from 'antd';
+import { updatePlotConfig } from '../../../../../../redux/actions/componentConfig';
+
+const HeatmapGroupBySettings = () => {
+  const dispatch = useDispatch();
+  const groupedTrack = useSelector((state) => state.componentConfig.interactiveHeatmap.config.groupedTrack);
```

Comment:
nice idea to put it in the plot config. Will that get saved in dynamo similarly to the plot config for plots in plots and tables?

Reply:
No, it wouldn't. I don't think that's a good idea for a component that a user would modify a lot. The save action never gets called.